osidb/mixins.py

```diff
@@ -279,7 +279,7 @@ def set_acl_write(self, *groups):
         self.acl_write = acls
         return acls
 
-    def set_public(self):
```

Comment:
I'm not sure I agree with this -- `set_public` is a low-level method that does one job: change an ACL-enabled object to public ACLs regardless of its current state, said state can even already be public for all it cares.

It's the caller's responsibility to ensure that only objects that should be public are passed to this method, this method is unaware of business rules and should stay unaware IMO.